shared/settings/landing/container.js

```diff
@@ -12,12 +12,13 @@ const mapStateToProps = state => {
   const {emails} = state.settings.email
   const {rememberPassphrase} = state.settings.passphrase
   let accountProps
-  if (emails) {
+  if (emails && emails.count() > 0) {
     let emailProps = {}
-    if (emails.length) {
+    const email = emails.get(0)
```

Comment:
FWIW, I think just do `emails && emails.first()` would also work. It just gives you `undefined` if the thing is empty.